src/runtime/callbacks-test.js

```diff
@@ -133,6 +133,48 @@ describes.sandboxed('Callbacks', () => {
     expect(callbacks.hasOffersFlowRequestCallback()).to.be.true;
   });
 
+  describe('warnings', () => {
+    let warnFn;
+
+    beforeEach(() => {
+      warnFn = sandbox.spy(self.console, 'warn');
+    });
+
+    afterEach(() => {
+      warnFn.restore();
```

Comment:
I don't think this is needed, since we have a global teardown function that restores all fakes: https://github.com/subscriptions-project/swg-js/blob/main/test/describes.js#L196

Reply:
Nice catch! That is great